src/binary-reader-objdump.cc

```diff
@@ -572,6 +583,7 @@ Result BinaryReaderObjdumpDisassemble::OnLocalDecl(Index decl_index,
 void BinaryReaderObjdumpDisassemble::LogOpcode(size_t data_size,
                                                const char* fmt,
                                                ...) {
+  assert(in_function_body);
```

Comment:
why can an opcode not be logged if it's in an init expr?

Reply:
This entire class (BinaryReaderObjdumpDisassemble) has the job of disassembly the code section, so it ignores instruction that appear outside the code section.   I can add a comment.   We have another class in this file that does log the instruction outside of the code section.

Reply:
makes sense, and a comment would be good.